pfcpiface/up4.go

```diff
@@ -647,9 +761,269 @@ func (up4 *UP4) updateTunnelPeersBasedOnFARs(fars []far) error {
 	return nil
 }
 
+func getMeterConfigurationFromQER(mbr uint64, gbr uint64) *p4.MeterConfig {
+	defaultBurstDurationMs := 10
+	logger := log.WithFields(log.Fields{
+		"GBR (Kbps)":         gbr,
+		"MBR (Kbps)":         mbr,
+		"burstDuration (ms)": defaultBurstDurationMs,
+	})
+	logger.Debug("Converting GBR/MBR to P4 Meter configuration")
+
+	cbs := calcBurstSizeFromRate(gbr, uint64(defaultBurstDurationMs))
+	pbs := calcBurstSizeFromRate(mbr, uint64(defaultBurstDurationMs))
+
+	var cir, pir uint64 = 0, 0
+	if mbr != 0 || gbr != 0 {
+		/* MBR/GBR is received in Kilobits/sec.
+		   CIR/PIR is sent in bytes */
+		cir = maxUint64((gbr*1000)/8, 0)
```

Comment:
We may need to revise this and send 1 instead of 0 as the default value.

Reply:
The rationale is to be consistent with BESS-UPF?

Reply:
No, we may need that if we want to use stratum_bmv2 in integration tests. Stratum will reject 0 as default value.